.renovaterc.json

```diff
@@ -12,10 +12,11 @@
       "separateMinorPatch": true
     },
     {
+      "matchDepNames": ["github/codeql-action"],
       "matchPackagePatterns": [
         "^ruby\\/setup-ruby$"
       ],
```

Comment:
Two questions:
1. These rules are disjunctive (either, but not both need to match)?
2. Since `ruby/setup-ruby` is also a GitHub action, can it also be specified using `matchDepNames` (or, if we want to consistently keep using `matchPackagePatterns`, can `github/codeql-action` be also expressed as such)?

Reply:
1. Yes, for example; it opened the CodeQL PR while the PSM is already up-to-date (in my local testing setup).
2. I see there is no entry for Ruby yet in our config, so I can file a PR there to add it.

(I was wondering why we didn't get any updates, but we are still on the latest: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby).

Reply:
Verified with local testing that the `depName` is `ruby/setup-ruby` 😄.

Added a commit to use the deps in a similar way.